src/dictionary/repo.ts

```diff
@@ -55,11 +56,12 @@ export const schemaRepo: SchemaRepository = {
 	get: async (
 		name: string,
 		versionToIgnore?: string,
+		requestedVersion?: string,
```

Comment:
Allows us to use standard `get` behaviour instead of only `versionToIgnore`

Reply:
do we use `versionToIgnore` anywhere? cause I don't know why this was included, or what the purpose is.

Reply:
... no, not that I can find in the codebase
It's an odd one for sure

Reply:
if we never use that argument then i'd suggest we can remove it and simplify our logic.